tests/test_exposure.py

```diff
@@ -773,6 +800,24 @@ def _getExposureCenter(self, exposure):
         """
         return exposure.getWcs().pixelToSky(lsst.geom.Box2D(exposure.getBBox()).getCenter())
 
+    def _getValidCorners(self, imageBox, cutoutBox):
+        """Return the corners of a cutout that are constrained by the original image.
+
+        Parameters
+        ----------
+        imageBox: `lsst.geom.Extent2I`
+            The bounding box of the original image.
+        cutoutBox : `lsst.geom.Box2I`
+            The bounding box of the cutout.
+
+        Returns
+        -------
+        corners : iterable of `lsst.geom.Point2I`
+            The corners that are drawn from the original image.
+        """
+        # TODO: prefer set, but need to fix DM-9938 first
```

Comment:
I think you'd also need to make `Box` immutable in order to be able to use it (safely) in a `set`).

Reply:
I'll remove the comment for now (it was referring to the wrong issue anyway).